migrations/sql/V2021.05.25.15.10__create_mine_work_information_table.sql

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE IF NOT EXISTS mine_work_status (
+  mine_work_status_code varchar PRIMARY KEY,
```

Comment:
Usually, our code-tables have PK of varchar(3), is there any reason to have these without any limit?

Reply:
No reason, I guess it was an oversight. I could change them to be 3 chars only.